src/components/Tooltip/TooltipTypes.d.ts

```diff
@@ -43,7 +43,7 @@ export interface ITooltip {
   id?: string
   variant?: VariantType
   anchorId?: string
-  wrapper?: WrapperType
```

Comment:
possible `undefined` removed because there's no way to use Tooltip component directly without Controller and Controller always passes a `wrapper` to Tooltip